source/includes/api/tables/progress-response.rst

```diff
@@ -128,3 +128,71 @@
 
        - When ``mongosync`` starts up, this field returns ``null`` until
          ``mongosync`` identifies the coordinator.
+
+   * - ``verification``
+     - document
+     - Describes the verification process on the source and destination clusters.
```

Comment:
Clarify what " Describes the verification process" means.

Reply:
Fixed